packages/react/components/table/src/TableRow/TableRow.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { chain } from '@react-aria/utils';
+import { cx } from '@project44-manifest/react-styles';
+import type { ForwardRefComponent } from '@project44-manifest/react-types';
+import { useTableContext } from '../Table.context';
+import { StyledTableRow } from './TableRow.styles';
+import type { TableRowElement, TableRowProps } from './TableRow.types';
+
+export const TableRow = React.forwardRef((props, forwardedRef) => {
+  const {
+    as,
+    children,
+    className: classNameProp,
+    css,
+    onMouseEnter: onMouseEnterProp,
+    onMouseLeave: onMouseLeaveProp,
+    ...other
+  } = props;
+
+  const { onMouseEnter, onMouseLeave, showHover } = useTableContext();
+
+  const [isHovered, setIsHovered] = React.useState(false);
+
+  const handleMouseEnter = React.useCallback(() => {
+    if (!showHover) return;
+
+    setIsHovered(true);
+  }, [setIsHovered, showHover]);
+
+  const className = cx('manifestui-table-row', classNameProp);
```

Comment:
reserving the change of `manifestui-table-row` to `manifest-table-row` for its own PR with major version bump